### DISCUSSION TITLE
Install (some) drivers during the installation process of Linux Mint?

### DISCUSSION THREAD

**Discussion by anaximeno** [linuxmint/discussions#33] [Ideas]
### Problem

When installing Linux Mint on a computer that requires the installation of some sort of driver to work properly people tend to have a problem working around this. I heard reports of some people that gave up using the system because after installing Linux Mint their wifi adapter stopped working properly.

### Proposed Solution

It should be possible to install the most important drivers (at least the wifi one) during the installation process, this will avoid people not using the system because they don't know that a driver should be installed for the system to work correctly, especially when the problem is with the wifi driver, because it may be difficult for some people to find an available wired connection and do the process of installing the required drivers for the wifi to work.

### Impacts

This may improve the overall user experience while installing the system and increase user retention.
Things to take into account

Installing the drivers should be optional in some fashion, as we know, not all users like these sorts of things to be installed automatically.

**Comment by jeremyb31** [comment]
I think you are referring to the Broadcom proprietary wifi driver.  If it is installed before installing Mint, it is needed to check the "Install third party drivers and software" for the driver to be present when booting the install of Mint

**Reply by anaximeno** [reply]
That's strange, I always check that option but the Broadcom driver never gets installed by default on my computer, and I've installed Mint many times.

I'll do more research on that when having more free time.

**Reply by anaximeno** [reply]
No, it doesn't get installed with that option, I generally have to install the driver through the driver manager after completing the installation (even after enabling the "Install third-party drivers and software" option). I found [this old article](https://community.linuxmint.com/tutorial/view/1643) by Clem showing how to install Mint on a Mac (which is the type of computer I have), and it seems the process could be done by using the bootable drive after the installation.

**Comment by anaximeno** [comment]
A solution could be to chroot the driver manager into the newly installed system and open the window immediately after completing the installation, allowing users to install the required drivers.
